examples/plain-javascript/README.md

```diff
@@ -49,6 +52,17 @@ data?](../../doc/cldr.md).
 
 No action needed here.
 
+**3. Globalize dist files**
+
+To run the demo you'll need Globalize to be built in order that the `../../dist` 
+directory has been populated.  If you haven't already done this then go to the root
+and execute the following:
+
+```
+npm install
+bower install
+grunt
+```
```

Comment:
Instead of duplicating the instructions here, I suggest we link the details with something in these lines: `If you haven't already done this then [install the development external dependencies](../../README.md#install-development-external-dependencies) and [build the distribution files](../../README.md#build).`